src/Cake.Testing.Xunit/WindowsFact.cs

```diff
@@ -1,29 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Cake.Core;
-using Cake.Core.Polyfill;
-using Xunit;
-
-namespace Cake.Testing.Xunit
-{
-    public sealed class WindowsFact : FactAttribute
```

Comment:
As this is a public assembly ( https://www.nuget.org/packages/Cake.Testing.XUnit )
and this is a breaking change should we obsolete first?

Reply:
@devlead you're right, I haven't thought of that. I changed the `WindowsFact` to `WindowsFactAttribute` out of habit.

The docs say
> By convention, attribute classes are named with a suffix of Attribute. An *attribute_name* can either include or omit this suffix.

So I'd suggest I revert the name change. This would leave only the change in inheritance, which isn't breaking, right?

Reply:
why not make WindowsFact inherit from WindowsFactAttribute and obsolete WindowsFact with a warning.

Reply:
No joy. Having an attribute called `WindowsFact` and one called `WindowsFactAttribute` results in

>   [CS1614] 'WindowsFact' is ambiguous between 'WindowsFact' and 'WindowsFactAttribute'. Either use '@WindowsFact' or explicitly include the 'Attribute' suffix. 

So, it seems while this keeps ABI compatibility, it breaks the API.

Reply:
Reasonable, and then we can rename it in 3.0 (add an issue for that).

Reply:
renamed